src/web/handlers/GestureHandler.ts

```diff
@@ -751,60 +686,130 @@ export default abstract class GestureHandler implements IGestureHandler {
     this.delegate.destroy(this.config);
   }
 
-  //
-  // Getters and setters
-  //
+  public getTrackedPointersID(): number[] {
+    return this.pointerTracker.getTrackedPointersID();
+  }
```

Comment:
Is this intentional? I _get_ that `getTrackedPointersID` is just forwarding a function with the same name, but I'm not sure why we can't make it a getter called `trackedPointersID`

Reply:
> Since we're already changing the name of the function, is it ok if we rename it to trackedPointerIDs?

Sure, we can

> I'm not sure why we can't make it a getter called trackedPointersID

Here we have to decide whether we want to have getters to values that are calculated, not stored in a private fields. It is already done `ScaleGestureDetector`, but I'm still not sure if we want that. 

On the other hand, i get that it doesn't look good that we have ES6 and Java-style getters 😞 

What's your stance on that, @j-piasecki?